cluster.yml

```diff
@@ -9,6 +9,8 @@ auth:
 # flannel
 # calico
 # canal
+#
+# if you are using calico on AWS, use the config option: calico_aws: true
```

Comment:
I feel aws should be a part of plugin options...not a top level. @galal-hussein @moelsayed what are your thoughts on that

Reply:
Oh, looked down the code, seems like it is a part of options. Can we rephrase the help message? Something like "for plugin calico, put aws=true to network/options field", otherwise it not clear what "config options" implies. Lets also rename calico_aws to aws as calico is redundant in this case.